mdpow/tests/test_equilibration.py

```diff
@@ -0,0 +1,57 @@
+import mdpow.equil
+import tempfile
+import os
+import shutil
+import mdpow.run
+import gromacs.environment
+gromacs.environment.flags['capture_output'] = True
+
+class TestEquilibration(object):
+    sim_filename = ".simulation"
+    sims = {"water": mdpow.equil.WaterSimulation,  
+            "octanol":mdpow.equil.OctanolSimulation, 
+            "cyclohexane":mdpow.equil.CyclohexaneSimulation}
+    
+    def setup(self):
```

Comment:
Is `setup()` actually executed by **py.test**? According to [classic xunit-style setup](https://pytest.org/latest/xunit_setup.html) such methods have to be called differently (maybe `setup_class()`?) *or* the class must be derived from `unitest.TestCase`.

Apprently, the recommended way to do fixtures is [py.test fixtures](https://pytest.org/latest/fixture.html#fixture) ... but I find it rather convoluted.